test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeSystemArraycopyTests.java

```diff
@@ -60,6 +59,7 @@ public static class SomeIdentityClass2 implements SomeInterface {
 		}
 	}
 
+	@ImplicitlyConstructible
```

Comment:
Is there any difference between `SomaValueClass` and `SomePrimitiveValueClass` ?

Reply:
There is no difference. I was thinking it would be best to leave them as two different classes for now until we have javac null-restricted support, otherwise it will be difficult to know which local fields should be marked as null-restricted since there is no way of representing that currently. Looking again I think all the uses of SomePrimitiveValueClass are for arrays so it should be okay to combine these classes into one. What do you think?

Reply:
>  Looking again I think all the uses of SomePrimitiveValueClass are for arrays so it should be okay to combine these classes into one. What do you think?

This is SystemArraycopyTests, so we are dealing with arrays only here. I think combining them looks better to me. 
Also the arrays here are either null-restricted array or nullable array. We can replace `primitiveVtArraySrc/Dst` with `nullRestrictedVtArraySrc/Dst`. They are both null-restricted arrays.